app/renderer/components/navigation/urlBarIcon.js

```diff
@@ -111,7 +111,9 @@ class UrlBarIcon extends React.Component {
 
   onDragStart (e) {
     dndData.setupDataTransferURL(e.dataTransfer, this.props.location, this.props.title)
-    dndData.setupDataTransferBraveData(e.dataTransfer, dragTypes.TAB, this.props.activeFrame)
+    dndData.setupDataTransferBraveData(e.dataTransfer, dragTypes.TAB, Immutable.fromJS({
```

Comment:
that function turns it right back into js again. I think we should stick with regular js